<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="XeroAdd" tests="1" time="109.913751" failures="0" errors="1" skipped="0" assertions="1">
  <testcase name="test_xero_invoice" time="109.913751" assertions="1">
    <error type="Timeout::Error" message="Timeout::Error">
Timeout::Error (Timeout::Error)
C:/Ruby193/lib/ruby/1.9.1/net/protocol.rb:146:in `rescue in rbuf_fill'
C:/Ruby193/lib/ruby/1.9.1/net/protocol.rb:140:in `rbuf_fill'
C:/Ruby193/lib/ruby/1.9.1/net/protocol.rb:122:in `readuntil'
C:/Ruby193/lib/ruby/1.9.1/net/protocol.rb:132:in `readline'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:2562:in `read_status_line'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:2551:in `read_new'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1319:in `block in transport_request'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1316:in `catch'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1316:in `transport_request'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1293:in `request'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1286:in `block in request'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:745:in `start'
C:/Ruby193/lib/ruby/1.9.1/net/http.rb:1284:in `request'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/http/default.rb:83:in `response_for'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/http/default.rb:39:in `request'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/http/common.rb:40:in `call'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/bridge.rb:634:in `raw_execute'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/bridge.rb:612:in `execute'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/remote/bridge.rb:580:in `find_element_by'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/selenium-webdriver-2.41.0/lib/selenium/webdriver/common/search_context.rb:42:in `find_element'
XeroTest.rb:38:in `test_xero_invoice'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testcase.rb:691:in `run_test'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testcase.rb:431:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testsuite.rb:121:in `run_test'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testsuite.rb:53:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testsuite.rb:121:in `run_test'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/testsuite.rb:53:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunnermediator.rb:65:in `run_suite'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunnermediator.rb:44:in `block in run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunnermediator.rb:100:in `with_listener'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunnermediator.rb:40:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunner.rb:40:in `start_mediator'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunner.rb:25:in `start'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/ui/testrunnerutilities.rb:24:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/autorunner.rb:409:in `block in run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/autorunner.rb:465:in `change_work_directory'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/autorunner.rb:408:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit/autorunner.rb:59:in `run'
C:/Ruby193/lib/ruby/gems/1.9.1/gems/test-unit-2.5.5/lib/test/unit.rb:502:in `block (2 levels) in &lt;top (required)&gt;'    </error>
  </testcase>
  <system-out>
  </system-out>
  <system-err>
  </system-err>
</testsuite>
